src/pattern-library/components/patterns/prototype/TabbedDialogPage.tsx

```diff
@@ -207,6 +206,7 @@ function TabbedSharePanel() {
             variant="custom"
             title="Share annotations"
             onClose={() => setPanelOpen(false)}
+            transitionComponent={Slider}
```

Comment:
Without the fix in #1157 this introduces a regression with tab sequence and focus, but with the fix, it seems to work fine!

Reply:
The fix is now merged, so feel free to rebase 🙂